docs/docs/mdx/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: MDX
+---
+
+[MDX][mdx] is Markdown for the component era.
+It lets you write JSX embedded inside Markdown.
+It’s a great combination because it allows you to use Markdown’s often terse
+syntax (such as `# heading`) for your content and JSX for more advanced
+components.
+
+This is useful in content-driven sites where you want the ability
+to introduce components like charts or notifs without having to
```

Comment:
What are notifs?

Reply:
An obscure term for alert, I'll change it 😅